File: src/main/java/net/kenro/ji/jin/purescript/psi/PSElementType.java
package net.kenro.ji.jin.purescript.psi;


import com.intellij.psi.tree.IElementType;
import net.kenro.ji.jin.purescript.PSLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class PSElementType extends IElementType {
    public PSElementType(@NotNull @NonNls final String debugName) {
        super(debugName, PSLanguage.INSTANCE);
    }
}

File: src/main/java/net/kenro/ji/jin/purescript/psi/PSElementType.kt
package net.kenro.ji.jin.purescript.psi

import com.intellij.psi.tree.IElementType
import net.kenro.ji.jin.purescript.PSLanguage
import org.jetbrains.annotations.NonNls

class PSElementType(@NonNls debugName: String) :
    IElementType(debugName, PSLanguage.INSTANCE)